src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -34,154 +37,220 @@ type State = AsyncComponent['state'] & {
   relays: Array<Relay>;
 };
 
-const relaysMock: Array<Relay> = [
-  {
-    public_key: '1:bb:6e:af:66:b4:38:e0:62:83:62:15:22:7',
-    name: 'First key',
-    description: 'optional description for the key',
-    first_seen: '2020-02-07T15:17:00Z',
-    last_seen: '2020-02-07T15:17:00Z',
-    created: '2020-02-07T15:17:00Z',
-  },
-  {
-    public_key: '2:bb:6e:af:66:b4:38:e0:62:83:62:15:22:7',
-    name: 'Second key',
-    description: 'optional description for the key',
-    first_seen: '2020-02-07T15:17:00Z',
-    last_seen: '2020-02-07T15:17:00Z',
-    last_modified: '2020-02-07T15:17:00Z',
-    created: '2020-02-07T15:17:00Z',
-  },
-];
-
 class Relays extends AsyncComponent<Props, State> {
+  constructor(props: Props, context: AsyncComponent['context']) {
+    super(props, context);
+    this.handleOpenAddDialog = this.handleOpenAddDialog.bind(this);
+  }
+
   getDefaultState() {
     return {
       ...super.getDefaultState(),
-      relays: relaysMock,
+      relays: this.props.organization.trustedRelays,
     };
   }
 
-  // TODO(Priscila): activate the code below as soon as the endpoint is provided
-  // getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
-  //   return [['relays', `/organizations/${this.props.organization.slug}/relay-keys`]];
-  // }
+  setRelays(trustedRelays: Array<Relay>) {
+    this.setState({relays: trustedRelays});
+  }
 
-  // handleDelete = (id: Relay['public_key']) => () => {};
+  handleDelete(publicKey: Relay['publicKey']) {
```

Comment:
Do theese handle methods need to be arrow function too?

Reply:
I decided to make only the return method an arrow function because it already has the bind